types/chartjs-plugin-annotation/tslint.json

```
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-outside-dependencies": false
+  }
+}
```

```suggestion
{
  "extends": "dtslint/dt.json"
}
```
what about defaulting to 'default' settings? 
It obviously would work on CI as dependencies are being installed for CI builds,
on the local machine it's enough to run `npm i --no-package-lock` for each depenedency, here for `moment' in `types/chartjs-plugin-annotation` and for 'chart.js` in `types/chart.js`.
Could using this exclusion prevent us from catching future errors?

---

hi @peterblazejewicz 

I'm not really familiar with how the CI build works with type dependencies that are defined outside DefinetlyTyped, I just copied what I saw other type definitions doing when working with `moment`.

I looked at a few, including `chart.js` and all seemed to have this combination: `moment` in package.json, `no-outside-dependencies` in tslint.json.

I agree, if this works without setting it, then we should keep tslint.json minimal.

Let's try it and see if it works :)


PS: do you know if there are some docs on how we should work with external type dependencies? and when does [allowedPackageJsonDependencies.txt](https://raw.githubusercontent.com/microsoft/DefinitelyTyped-tools/master/packages/definitions-parser/allowedPackageJsonDependencies.txt) come into play?